lib/go-atscfg/remapdotconfig.go

```diff
@@ -202,6 +202,28 @@ func cachekeyArgsFor(configParamsMap map[string][]tc.Parameter, warnings *[]stri
 	return
 }
 
+// Returns any pre or post raw remap lines
```

Comment:
Nitpick: this breaks GoDoc. Symbol comments should be complete sentences starting with the symbol name and including punctuation, e.g. `// lastPrePostRemapLinesFor returns any pre or post raw remap lines.`